flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -444,7 +447,7 @@ public void testKeyedCEPOperatorNFAUpdateWithRocksDB() throws Exception {
             operator = CepOperatorTestUtilities.getKeyedCepOperator(true, new SimpleNFAFactory());
             harness = CepOperatorTestUtilities.getCepTestHarness(operator);
 
-            rocksDBStateBackend = new RocksDBStateBackend(new MemoryStateBackend());
+            rocksDBStateBackend = new EmbeddedRocksDBStateBackend(TernaryBoolean.TRUE);
```

Comment:
Should we use the constructor without any parameter, `new EmbeddedRocksDBStateBackend();` ?

Same for any other places